python_code_review/test_review.py

```diff
@@ -0,0 +1,18 @@
+class Test:
+	def __init__(self, message):
+		self.message = message
+
+	def set_message(self, text):
+		self.message = text
+
+
+print("hello world")
+
+computer = Test("Hello World")
+
+print(computer.message)
+
+computer.set_message("Leaving World")
+
+print(computer.message)
```

Comment:
This is another test message. Reply to this please

Reply:
Changed, please take another look